### PR TITLE
logtail: actually log panics

### PR DESCRIPTION
**Pull request by crawshaw**

### CODE REVIEW

File: logtail/logtail.go [hunk 1]
```diff
@@ -118,6 +118,7 @@ type Logger struct {
 	bo             *backoff.Backoff
 	zstdEncoder    Encoder
 	uploadCancel   func()
+	explainedRaw   bool
 
 	shutdownStart chan struct{} // closed when shutdown begins
 	shutdownDone  chan struct{} // closd when shutdown complete
```
Comment (bradfitz) [inline logtail/logtail.go:124]: typo: closed

Comment (josharian) [reply logtail/logtail.go:124]: will fix in separate commit

**Comment by bradfitz** [comment]
I was waiting on more changes from @crawshaw, to not replace stderr when running tailscaled interactively on the console. I want to see my panics to stderr.

**Comment by josharian** [comment]
Yes, very sorry about that.

**Comment by crawshaw** [comment]
On my todo list for tomorrow.

**Pull request merged**